app/femr/business/services/core/IPatientService.java

```diff
@@ -51,7 +51,7 @@
      * @return a service response that contains a PatientItem representing the patient that was created
```

Comment:
a comment describing the check parameter should be added here so people know what it is doing in the future